File: Sample Apps/Objective-C/Reference App/ViewControllers/ContainerViewController.m
//
//  ContainerViewController.m
//  Reference App
//
//  Created by Aaron Bratcher on 1/3/18.
//  Copyright © 2018 Liveperson. All rights reserved.
//

#import "ContainerViewController.h"
#import "Reference_App-Swift.h"

@interface ContainerViewController ()

@end

@implementation ContainerViewController

- (void)viewDidLoad {
    [super viewDidLoad];
	[[MessagingExperience shared] showConversationFor:self.referenceID name:self.title ?: @"" with:@"" in: self];
}

@end

File: Sample Apps/Swift/Reference App/ViewControllers/ContainerViewController.swift
//
//  ConsumerSettingsController.swift
//  CAO Test
//
//  Created by Aaron L. Bratcher on 11/14/17.
//  Copyright © 2017 Liveperson. All rights reserved.
//


import UIKit

class ContainerViewController: UIViewController {
	var referenceID = ""
	var initialUserText: String? = nil

	override func viewDidLoad() {
		MessagingExperience.shared.showConversation(for: referenceID, name: title ?? "", with: initialUserText, in: self)
	}
}

